packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2217,6 +2219,13 @@ function commitMutationEffects(root: FiberRoot, renderPriorityLevel) {
       if (current !== null) {
         commitDetachRef(current);
       }
+      if (enableScopeAPI) {
+        // This is a temporary solution that allows us to transition away
```

Comment:
super nit: Could we prefix these with "TODO" so they stand out more than a regular comment?